pkg/controller/backup/backup_controller.go

```diff
@@ -230,6 +230,10 @@ func (c *Controller) updateBackup(cur interface{}) {
 		return
 	}
 
+	if newBackup.Spec.Mode == v1alpha1.BackupModeLog && newBackup.Spec.LogSubcommand == string(v1alpha1.LogStartCommand) && v1alpha1.IsBackupPaused(newBackup) {
+		newBackup.Spec.LogSubcommand = string(v1alpha1.LogResumeCommand)
```

Comment:
I don't know why add this hack logic. We shouldn't modify the `spec` fields even the user inputs wrong value.

Reply:
deleted